version 0.4.0 is missing types

The package.json still has `"typings": "types/index.d.ts",` but that file is not in the npm package, which leads to errors like this:

```
$ tsc                    
src/index.ts:1:52 - error TS7016: Could not find a declaration file for module '@oruga-ui/theme-bulma'. '/home/marnold/work/commun/bulma-ovea/node_modules/@oruga-ui/theme-bulma/dist/bulma.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/oruga-ui__theme-bulma` if it exists or add a new declaration (.d.ts) file containing `declare module '@oruga-ui/theme-bulma';`

1 import { bulmaConfig as originalBulmaConfig } from '@oruga-ui/theme-bulma';
                                                     ~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error in src/index.ts:1
```

--- a/package.json
+++ b/package.json
@@ -6,7 +6,7 @@
   "main": "dist/bulma.js",
   "module": "dist/bulma.mjs",
   "unpkg": "dist/bulma.min.js",
-  "typings": "types/index.d.ts",
+  "types": "dist/types/bootstrap.d.ts",
   "sideEffects": [
     "*.css",
     "*.scss"

--- a/package.json
+++ b/package.json
@@ -6,7 +6,7 @@
   "main": "dist/bulma.js",
   "module": "dist/bulma.mjs",
   "unpkg": "dist/bulma.min.js",
-  "types": "dist/types/bootstrap.d.ts",
+  "types": "dist/types/bulma.d.ts",
   "sideEffects": [
     "*.css",
     "*.scss"

--- a/rollup.config.mjs
+++ b/rollup.config.mjs
@@ -34,7 +34,15 @@ const commonSassPluginOptions = {
 };
 
 const typescriptPluginOptions = {
-  sourceMap: false
+  sourceMap: false,
+  // skip type checking of declaration files
+  skipLibCheck:true,
+  // enabling declaration (.d.ts) emit
+  declaration: true,
+  //  decouple declaration files from actual transpiled JavaScript files
+  declarationDir: "dist/types",
+  // define included files
+  include: "src/plugins/**",
 };
 
 function createDirectoryIfDoesNotExist(filePath) {

--- a/src/plugins/bulma.ts
+++ b/src/plugins/bulma.ts
@@ -1,9 +1,10 @@
 import type { OrugaOptions, ComponentProps } from "@oruga-ui/oruga-next";
 import { isTrueish } from "@oruga-ui/oruga-next";
 
+// import theme style
 import "../assets/scss/bulma-build.scss";
 
-export const bulmaConfig: OrugaOptions = {
+const bulmaConfig: OrugaOptions = {
     field: {
         override: true,
         rootClass: "field",
@@ -430,3 +431,5 @@ export const bulmaConfig: OrugaOptions = {
         hoveredClass: "is-hovered",
     },
 };
+
+export { bulmaConfig, bulmaConfig as config };

--- a/src/shims-vue.d.ts
+++ b/src/shims-vue.d.ts
@@ -1,6 +1,0 @@
-/* eslint-disable */
-declare module '*.vue' {
-  import type { DefineComponent } from 'vue'
-  const component: DefineComponent<{}, {}, any>
-  export default component
-}

